app/components/IntegrationCard.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import { s, ellipsis } from "@shared/styles";
+import { ConfigItem } from "~/hooks/useSettingsConfig";
+import Button from "./Button";
+import Flex from "./Flex";
+
+type Props = {
+  integration: ConfigItem;
+};
+
+function IntegrationCard({ integration }: Props) {
+  const { t } = useTranslation();
+  return (
+    <Card>
+      <Flex justify="space-between" align="center">
+        <Flex align="center" gap={20}>
+          <integration.icon size={50} />
+          <Flex column gap={4}>
+            <Name>{integration.name}</Name>
+
+            {integration.isActive && (
```

Comment:
We use this style on e.g GitHub and Linear integration pages, can have similar here with the status dot I think and `textSecondary` – the lime green is impossible to read.

<img width="142" alt="image" src="https://github.com/user-attachments/assets/10fb80c9-d37e-47dc-8cf1-27f2bc862662" />